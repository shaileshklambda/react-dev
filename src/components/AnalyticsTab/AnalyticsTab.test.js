import React from "react";
import renderer from "react-test-renderer";
import Analyticstab from ".";

it("Analyticstab: default", () => {
  const component = renderer.create(<Analyticstab />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
