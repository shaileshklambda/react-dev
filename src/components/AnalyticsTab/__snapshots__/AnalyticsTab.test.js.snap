// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Analyticstab: default 1`] = `
<div
  className="formWrapper"
>
  <form
    className="filterBar analyticsFilterBar form-inline"
  >
    <div
      className="viewRadioButton"
    >
      <label>
        <input
          name="radio"
          type="radio"
        />
        <span
          className="labelText font12  fontLato"
        >
          Build View
        </span>
      </label>
      <label>
        <input
          name="radio"
          type="radio"
        />
        <span
          className="labelText font12  fontLato"
        >
          Test View
        </span>
      </label>
    </div>
    <div
      className="filterIcon"
    >
      <span>
        <img
          className=""
          src={
            Object {
              "ReactComponent": [Function],
              "__esModule": true,
              "default": "funnel.svg",
            }
          }
        />
      </span>
    </div>
    <div
      className="dropdownSelect form-group"
    >
      <select
        className="form-control"
        id="formControlsSelect1"
        placeholder="select"
      >
        <option
          value="select"
        >
          All Users
        </option>
        <option
          value="other"
        >
          ...
        </option>
        <option
          value="other1"
        >
          ...
        </option>
        <option
          value="other2"
        >
          ...
        </option>
        <option
          value="other3"
        >
          ...
        </option>
        <option
          value="other4"
        >
          ...
        </option>
        <option
          value="other5"
        >
          ...
        </option>
      </select>
    </div>
    <div
      className="timeLine font12"
    >
      <span>
        Today
      </span>
      <span
        className="active"
      >
        This Week 
      </span>
      <span>
        This Month
      </span>
    </div>
    <div
      className="settingsIcon"
    >
      <span>
        <img
          className=""
          src={
            Object {
              "ReactComponent": [Function],
              "__esModule": true,
              "default": "settings-icon.svg",
            }
          }
        />
      </span>
    </div>
  </form>
  <div
    className="addGraphButton"
  >
    <img
      className=""
      src={
        Object {
          "ReactComponent": [Function],
          "__esModule": true,
          "default": "plus.svg",
        }
      }
    />
    <span
      className="text font12"
    >
      Add graph
    </span>
  </div>
</div>
`;
